frontend/src/components/ConfigForm.vue

```diff
@@ -563,4 +589,8 @@ export default {
 .commonSvcBtn {
   min-height: 100px;
 }
+.underbutton {
+  padding-left: 20px;
+  margin-bottom: 15px;
+}
```

Comment:
No change required, but it may be good to consider using em/rem units instead of px when possible to allow for more reactive flexibility down the line.

Reply:
Yeah possibly would be good to change whole app px values over.